server/package.json

```diff
@@ -146,29 +141,23 @@
     "@types/ua-parser-js": "^0.7.36",
     "@types/validator": "^13.15.2",
     "@vitest/coverage-v8": "^3.0.0",
-    "canvas": "^3.1.0",
     "eslint": "^9.14.0",
     "eslint-config-prettier": "^10.1.8",
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-unicorn": "^60.0.0",
     "globals": "^16.0.0",
     "mock-fs": "^5.2.0",
-    "node-addon-api": "^8.3.1",
     "node-gyp": "^11.2.0",
     "pngjs": "^7.0.0",
     "prettier": "^3.0.2",
     "prettier-plugin-organize-imports": "^4.0.0",
-    "rimraf": "^6.0.0",
-    "source-map-support": "^0.5.21",
     "sql-formatter": "^15.0.0",
     "supertest": "^7.1.0",
     "tailwindcss": "^3.4.0",
     "testcontainers": "^11.0.0",
-    "tsconfig-paths": "^4.2.0",
     "typescript": "^5.9.2",
     "typescript-eslint": "^8.28.0",
     "unplugin-swc": "^1.4.5",
-    "utimes": "^5.2.1",
```

Comment:
library.e2e-spec.ts uses `utimes`

Reply:
It's still declared in the `package.json` for `e2e`